pyproject.toml

```diff
@@ -22,6 +22,12 @@ requires-python = ">=3.7"
 dependencies = [
     "flask>=2.2",
     "cachelib",
+    "redis",
+    "pymongo",
+    "python3-memcached",
+    "Flask-SQLAlchemy",
```

Comment:
Why are you adding all these runtime dependencies? Pretty much nobody will need them all at the same time. Those should almost certainly be behind extras.

Reply:
If the flask-session is to support the different interfaces but not include those as dependencies then it will not be able to run as configured. Since readthedocs state it, I will remove it from the PR, however, including it in someway might be a nice thing for a user wanting to use the interface in question.